app/Jobs/ProcessMediaWikiJobsJob.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Support\Facades\Log;
+use Maclof\Kubernetes\Client;
+use Maclof\Kubernetes\Models\Job as KubernetesJob;
+
+class ProcessMediaWikiJobsJob implements ShouldQueue, ShouldBeUnique
+{
+    use InteractsWithQueue, Queueable;
+
+    private string $wikiDomain;
+
+    public function __construct ( string $wikiDomain )
+    {
+        $this->wikiDomain = $wikiDomain;
+    }
+
+    public function uniqueId(): string
+    {
+        return $this->wikiDomain;
```

Comment:
This makes sure only a single job per wiki domain is ever enqueued. If the scheduler pushes another job for the same wiki onto the queue, it will be dropped instantly.